src/main/java/net/neoforged/neoforge/common/ModConfigSpec.java

```diff
@@ -1000,6 +1000,10 @@ private Range(Class<V> clazz, V min, V max) {
             this.max = max;
         }
 
+        public static Range<Integer> of(int min, int max) {
```

Comment:
Should we validate that `min <= max`?

Reply:
That check already exists...at a weird location. I moved it into the constructor.